app/views/report_mailer/tables_report.html.erb

```diff
@@ -35,7 +35,7 @@
         <tr>
           <% row.each do |cell| %>
             <td <%= cell.is_a?(Numeric) ? 'class=table-number' : nil %> >
-              <% if cell.is_a?(Float) && report.float_as_percent? %>
+              <% if cell.is_a?(Float) && report.float_as_percent? && cell.finite? && cell.between?(0.0, 1.0) %>
```

Comment:
I flip-flopped on this. I think sometimes things like `-50%` or `150%` are valid! For things like growth/week-over-week, etc. The `finite?` check is great, big fan of that.

But this is also internal and we can fix it again later if needed. Luckily the attached CSVs, XLSX just contain the "raw" data

```suggestion
              <% if cell.is_a?(Float) && report.float_as_percent? && cell.finite? %>
```